modules/smart-agent_health-checker/conf/00-health-checker-value.yaml

```diff
@@ -0,0 +1,14 @@
+module: health-checker
+name: value
+id: health_checker_value
+
+transformation: true
```

Comment:
I would recommend to either define a transformation or a lasting function to avoid alert flapping.

Reply:
done